hw3/02-charts.js

```diff
@@ -1,45 +1,113 @@
 const backgroundColors = [
-  'rgba(54, 162, 235, 0.8)',
```

Comment:
Another set of quotes that were probably automatically updated by formatter, single quotes are recommended by AirBnB so I would watch out for quotes getting updated in JS files.